hieroplatform/src/main/java/org/hiero/utils/Randomness.java

```diff
@@ -0,0 +1,34 @@
+package org.hiero.utils;
+
+import org.apache.commons.math3.random.MersenneTwister;
+
+public class Randomness {
```

Comment:
This class should be documented carefully, since everyone who is using random number generators is expected to use this class. You should start by declaring an interface which has the last three methods in the signature. Then the class can extend the signature. In the README.md you should make it clear that people should never use java.util.Random, but instead this class.

Also, we should figure out the correct protocol to use this class. Who should initialize the seed? It could be initialized once, at application start-up time, or every time you make a call? Also, note that the hiero service is shared among many potential clients, so the random number generator will also be shared.

We should add a proper API for initializing.

Maybe the class should have only static methods? It's kind of weird to say:

```
Randomness i = Randomness.getInstance();
int x = r.getNext();
```

How about just

```
int x = Randomness.getNext();
```